_sub/compute/helm-goldpinger/main.tf

```diff
@@ -1,7 +1,7 @@
 resource "helm_release" "goldpinger" {
   name          = "goldpinger"
   chart         = "goldpinger"
-  repository    = "https://charts.helm.sh/stable"
+  repository    = "https://okgolove.github.io/helm-charts/"
```

Comment:
Make sure to update the Helm exporter vars when this is released.

Reply:
Thanks